### ISSUE TITLE
Unable to import Events and ImageAddedEvent 

### ISSUE THREAD

**Issue by AshwinNema** [promosis/file-upload-with-preview#82]
While reading documentation I found that we can add event listener to the window object for the events happening. Here is one of the snippet given in documentation 

```
import { Events, ImageAddedEvent } from 'file-upload-with-preview';

window.addEventListener(Events.IMAGE_ADDED, (e: Event) => {
  const { detail } = e as unknown as ImageAddedEvent;

  console.log('detail', detail);
});
```

The issue I am facing is that I am unable to import Events as they are not exported from package.json.

**Proposed Solution**

We can add it in exports in package.json like 

`"./dist/index":"./dist/index.d.ts”`

exports after adding above code will look like 

```
"exports": {
    ".": {
      "import": "./dist/file-upload-with-preview.js",
      "require": "./dist/file-upload-with-preview.cjs",
      "types": "./dist/index.d.ts"
    },
    "./dist/style.css": "./dist/style.css",
    "./dist/index":"./dist/index.d.ts"
  }
```

and if we want to import we can import like 

`import { Events, ImageAddedEvent } from 'file-upload-with-preview/dist/index';`

**Comment by AshwinNema** [comment]
Hi @johndatserakis ,

Can I create PR for this issue? Is there any better approach we can fix this?

**Comment by AshwinNema** [comment]
Alternatively we can also add it in import script like 

` "import": "./dist/file-upload-with-preview.js,./dist/index.d.ts"`

**Comment by johndatserakis** [comment]
Hi there @AshwinNema.

I just tested this here in [this codesandbox](https://codesandbox.io/p/sandbox/file-upload-with-preview-package-example-forked-lpr3j9?file=%2Fsrc%2Findex.ts%3A1%2C1-16%2C41) and I am able to import `Events` and `ImageAddedEvent` just fine. The code is below:

```ts
import {
  Events,
  FileUploadWithPreview,
  ImageAddedEvent,
} from "file-upload-with-preview";
import "file-upload-with-preview/dist/style.css";

window.addEventListener(Events.IMAGE_ADDED, (e: Event) => {
  const { detail } = e as unknown as ImageAddedEvent;

  console.log("detail", detail);
});

const firstUpload = new FileUploadWithPreview("myFirstImage");

console.log("firstUpload", firstUpload);
```

In looking at the exports, everything seems fine to me. I'd double check you have the latest version - `v6.0.2`.

Can you please share a repro or codesandbox showing the issue? I don't think any changes are needed here.

**Comment by AshwinNema** [comment]
Hi @johndatserakis ,

i checked it once again at my end. I am able to import Events and ImageAddedEvent at my end but the issue I am facing is that in typescript it is showing me red lines when I try to import these 

`Module '"file-upload-with-preview"' has no exported member 'Events’.`

I think typescript types have to be added but they are already there and making changes in import script like below resolves this issue

` "import": "./dist/file-upload-with-preview.js,./dist/index.d.ts"`

**Comment by AshwinNema** [comment]
I am using this npm in Next.js with typescript. I am attaching github link

https://github.com/AshwinNema/file-upload-with-preview

**Comment by AshwinNema** [comment]
Here is the [codesandbox](https://codesandbox.io/p/live/98976d58-c524-4464-88d1-8a2a29187534) link

**Comment by AshwinNema** [comment]
Hi @johndatserakis , I am using this package with next.js. Is there any better way to get type support for Events and ImageAddedEvent? Although I have found a workaround for myself but getting type support is the best approach I believe

**Comment by johndatserakis** [comment]
Thank you, Ashwin. I think I see the issue. I will hopefully have a PR up tonight fixing this. Will ping when ready.

**Comment by johndatserakis** [comment]
Alrighty, this has been fixed and is live in v6.1.2.

In summary, there are some `tsconfig.json` `moduleResolution` settings that evidently could get confused with my directory naming. I've updated the `exports` entry in the `package.json` to correctly point to the lib's index, which solves the issue and is the correct connection. I gather that this requirement was less strict in the past.

Thanks again for mentioning.
